### ISSUE TITLE
Opening url in snap?

### ISSUE THREAD

**Issue by shamiv** [alexmurray/emacs-snap#62]
I see that others have been facing similar issues, but the solutions haven't worked for me yet: how do I make sure that urls that I open in the emacs snap open in Firefox?

**Comment by alexmurray** [comment]
Hi

I'd be happy to try and help but I need some more details - can you provide some more details of your system / emacs configuration? In particular, what OS are you running and what desktop environment? Is firefox configured as the default application for web browsing? Have you customised any of the `browse-url` related variables in emacs?

Also where are URLs currently being opened when you click on them in emacs?

More specifically, what does the following command output for you?

```
xdg-settings get default-web-browser
```

and what is the value of `browse-url-browser-function` in emacs?

**Comment by shamiv** [comment]
Sorry, of course.

- I'm on Ubuntu 22.04
- Gnome 42.5
- Spacemacs 0.999.0@28.2
- Snap Emacs 28.2
- "Firefox Web Browser" is the default system browser
- Output of `xdg-settings get default-web-browser` is `firefox.desktop`
- The output of `browse-url-browser-function` is `browse-url-default-browser`
- I have not customized any browse-url related variables.

At this point urls do not open at all. So also not in a new window of Firefox or anything.

**Comment by alexmurray** [comment]
Hmm so there is nothing that jumps out to me from the above that seems in indicate any problem - out of interest, what happens if you run:

```
gtk-launch $(xdg-settings get default-web-browser)
```

Is firefox launched then?

**Comment by alexmurray** [comment]
@shamiv have you had a chance to follow up on this? Can you please try the suggestion I posted above and let me know what happens.

**Comment by shamiv** [comment]
Hi @alexmurray sorry for not getting back. I have moved to a completely different set-up (from Spacemacs to Vanilla Emacs) and everything has changed. Also I'm not using the snap anymore (I built Emacs 30). Everything seems to be working now.

If at some point someone else ends up struggling with this I can see if I can replicate the problem, but for now I would say let's leave it? 

(But thanks for taking this up!)
